src/SourceBrowser/src/SourceIndexServer/Program.cs

```diff
@@ -10,11 +11,19 @@ public static void Main(string[] args)
             BuildWebHost(args).Run();
         }
 
+        public static ILogger Logger { get; set; }
+
         public static IHost BuildWebHost(string[] args) =>
             Host.CreateDefaultBuilder(args)
                 .ConfigureWebHostDefaults(
                     builder => { builder
                         .UseStartup<Startup>(); })
+                .ConfigureLogging(logging =>
+                        {
+                            logging.ClearProviders();
+                            logging.AddConsole();
+                            logging.AddAzureWebAppDiagnostics();

```

Comment:
Where do these go? Are they going into the AppService logging somewhere in the portal?

Reply:
They go into the portal, as long as the portal is configured for it.